.github/workflows/release.yml

```diff
@@ -12,6 +12,28 @@ env:
   SCCACHE_VERSION: v0.4.1
 
 jobs:
+  generate-changelog:
+    name: Generate changelog
+    runs-on: ubuntu-latest
+    outputs:
+      release_body: ${{ steps.git-cliff.outputs.content }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Update config
+        shell: bash
+        run: sed -E -i "s/\s+\#\s(.*)\s\#\sreplace issue numbers/\\t\1/g" cliff.toml
```

Comment:
What does this `sed` do? Can we do it once?

Reply:
This will put the links in the body of the release. But if we generate a changelog inside the repository like gear/CHANGELOG.md -> PR numbers will be replaced by github.